pkg/daemon/client.go

```diff
@@ -173,7 +173,7 @@ func WaitUntilSocketExisted(sock string) error {
 
 		return nil
 	},
-		retry.Attempts(20), // totally wait for 2 seconds, should be enough
+		retry.Attempts(200), // totally wait for 20 seconds, should be enough
```

Comment:
Do we have to wait so long?

Reply:
How about 5s or 10s? In any case, 2s is too short and may not get the `nydusd' startup status right.

Reply:
What worries me is why nydusd does not start its HTTP server after 2 seconds. It should be long enough. Extend the wait time is not a solution to the inconsistency problem. We should allow nydusd's startup failure and handle the inconsistency of metrics counting

Reply:
But it's fine to expand the wait time for the existence of the UNIX socket. But we still have to handle the inconsistency condition. Let's merge this PR in the first place and raise another PR to handle the inconsistency once and for all.
Let's keep the issue https://github.com/containerd/nydus-snapshotter/issues/392 open to trace it. 10 seconds sounds good to me  🤗

Reply:
Well, I updated it.